packages/libs/sdk/src/api/graphql/queries/ethereum/mainnet/transactions/getBySearch.ts

```diff
@@ -3,7 +3,7 @@ import { TransactionsBySearch } from '../../../fragments/getTransactionsBySearch
 
 export const EthMainnetTransactionsBySearch = gql`
   query EthMainnetTransactionsBySearch(
-    $filter: TransactionsFilterInput!,
+    $filter: TransactionsFilterInput,
```

Comment:
I had this filter as required, but it doesn't need to be actually passed in (this is for the `transactions.getAll()` function)